pkg/controller/certificates/certificate_controller.go

```diff
@@ -109,8 +111,9 @@ func (cc *CertificateController) Run(ctx context.Context, workers int) {
 	defer utilruntime.HandleCrash()
 	defer cc.queue.ShutDown()
 
-	klog.Infof("Starting certificate controller %q", cc.name)
-	defer klog.Infof("Shutting down certificate controller %q", cc.name)
+	logger := klog.FromContext(ctx)
+	logger.Info("Starting certificate controller", "certificateControllerName", cc.name)
```

Comment:
```suggestion
	logger.Info("Starting certificate controller", "controllerName", cc.name)
```

might be enough. Or just `name`?

Reply:
Updated.